### PR TITLE
tutorials: add tutorial on debugging tips

### PR DESCRIPTION
**Pull request by haircommander**
Signed-off-by: Peter Hunt <pehunt@redhat.com>

#### What type of PR is this?

/kind documentation
#### What this PR does / why we need it:
so I don't forget how to do this every time anymore :smile: 
#### Does this PR introduce a user-facing change?

```release-note
none
```

### CODE REVIEW

File: tutorials/debugging.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Debugging CRI-O
```
Comment (saschagrunert) [inline tutorials/debugging.md:1]: Let's link this document from the README.md

Comment (haircommander) [reply tutorials/debugging.md:1]: updated!

**Review by saschagrunert — APPROVED** [review]
LGTM

**Pull request merged**